sites/partners/pages/application/[id]/index.tsx

```diff
@@ -31,6 +31,10 @@ import { DetailsHouseholdIncome } from "../../../src/applications/PaperApplicati
 import { DetailsTerms } from "../../../src/applications/PaperApplicationDetails/sections/DetailsTerms"
 import { Aside } from "../../../src/applications/Aside"
 
+export const getDetailFieldString = (listingString: string) => {
```

Comment:
Could we maybe move this over to `src` somewhere so it's not just hanging off the page file?

Reply:
can do 👍